src/current/cockroachcloud/upgrade-cockroach-version.md

```diff
@@ -0,0 +1,54 @@
+
+    A major version has two types of patch releases: a series of **testing releases** followed by a series of **production releases**. A major version’s initial production release is also known as its GA release. In the lead-up to a new major version's GA release, a series of Testing releases may be made available to CockroachDB {{ site.data.products.advanced }} as Pre-Production Preview releases for testing and validation. Testing releases are intended for testing and experimentation only, and are not qualified for production environments or eligible for support or uptime SLA commitments. If a cluster is upgraded to a Pre-Production Preview patch release, it will be automatically upgraded to subsequent patch releases within the major version, including newer Pre-Production Preview releases, the initial GA release, and subsequent Production patches.
+
+To learn more about CockroachDB major versions and patches, refer to the [Releases Overview]({% link releases/index.md %}#overview).
+
+## Upgrades and maintenance windows
+
+If you have [configured a maintenance window]({% link cockroachcloud/advanced-cluster-management.md %}) for a CockroachDB {{ site.data.products.advanced }} cluster, automatic patch upgrades are applied during the maintenance window. Major-version upgrades must be initiated manually.
+
+{{site.data.alerts.callout_info}}
+Maintenance operations that are critical for cluster security or stability may be applied outside of the maintenance window, and upgrades that begin in a maintenance window may not always be completed by the end of the window.
+{{site.data.alerts.end}}
+
+Refer to [Cloud Upgrade Policy]({% link cockroachcloud/upgrade-policy.md %}).
+
+## Perform a major-version upgrade
+
+This section shows how to perform a major-version upgrade for a cluster in CockroachDB {{ site.data.products.cloud }}. Patch upgrades within a cluster's major version are applied automatically, and no action is required.
+
+1. Verify the cluster's current major version, and which versions it can be upgraded to:
+    1. Sign into [CockroachDB {{ site.data.products.cloud }}](https://cockroachlabs.cloud).
+    1. From the **Clusters** page, find the cluster by name. If the cluster is in a folder, click the name of the folder to view its descendants. The cluster's major version and patch are shown in the **Version** column.
+1. Check which upgrades are available to the cluster, if any.
+
+    Beginning with v24.1, major versions that end in `.1` and `.3` are required Regular releases, which can be upgraded to either the the next Innovation release or the next Regular release. For example, a cluster running v24.1 (a Regular release) can be upgraded directly either v24.2 (the next Innovation release) or v24.3 (the next Regular release). A cluster running v24.2 must be upgraded to v24.3.
```

Comment:
.1 and .3 are the Regular releases only for 24.x. https://www.cockroachlabs.com/docs/releases/#recent-releases

As of 25.1 we plan for all .2 and .4 to be the Regular releases. https://www.cockroachlabs.com/docs/releases/#upcoming-releases

```suggestion
    Beginning with v24.1, major versions alternate in type between Regular releases, which are required, and Innovation releases, which can be skipped.
    
    From a Regular release, you can upgrade to the next major release (an Innovation release) or the subsequent major release (another Regular release). From an Innovation release, you must upgrade to the next Regular release.
    
    These releases also come with different support periods.
    
    For details, refer the [CockroachDB Cloud Support Policy]({% link cockroachcloud/upgrade-policy.md %}#cockroachdb-cloud-support-policy).
```